<?xml version="1.0"?>
<launch>
  <node name="tubaf_haf_client" pkg="tbf_gripper_autonomy" type="haf_client.py" output="screen">
    <param name="haf_server_name" value="haf_server" />
    <rosparam file="$(find tbf_gripper_autonomy)/config/std_params.yaml" command="load" />
  </node>
</launch>